diff --git a/dep/sqlpp11/include/sqlpp11/result.h b/dep/sqlpp11/include/sqlpp11/result.h
index 527a6e0..09de19f 100644
--- a/dep/sqlpp11/include/sqlpp11/result.h
+++ b/dep/sqlpp11/include/sqlpp11/result.h
@@ -117,6 +117,16 @@ namespace sqlpp
     {
       _result.next(_result_row);
     }
+
+    size_t size() const
+    {
+      return _result.size();
+    }
+
+    operator bool() const
+    {
+      return !empty();
+    }
   };
 }
 
